dev-docs/bidders/exadsadserver.md

```diff
@@ -0,0 +1,484 @@
+| `bidfloor`      | optional |  Minimum bid for this impression (CPM) / click (CPC) and account currency           | `0.00000011`    | `float` |
+| `bidfloorcur`      | optional |  Currency for minimum bid value specified using ISO-4217 alpha codes           | `'EUR'`    | `string` |
+| `bcat`      | optional |  Blocked advertiser categories using the IAB content categories           | `['IAB25', 'IAB7-39','IAB8-18','IAB8-5','IAB9-9']`    | `string array` |
+| `badv`      | optional |  Block list of advertisers by their domains            | `['first.com', 'second.com']`    | `string array` |
+| `native.plcmtcnt`      | optional |   the number of identical placements in this Layout             | `4`    | `integer` |
+| `assets`      | required |   To see the specific paragraph for more info             | `{}`    | `object` |
+| `dsa`      | optional |   DSA transparency information. To see paragraph for more info           | `{ dsarequired: 3, pubrender: 0, datatopub: 2 }`    | `object` |
+| `endpoint`      | required |   EXADS endpoint (URL)            | `https://your-ad-network.com/rtb.php`    | `string` |
+
+##### Assets
+
+* **assets (title)**
+  * **id** - unique asset ID, assigned by exchange. Typically a counter for the array (integer):
+        *1 - image asset ID
+        * 2 - title asset ID
+        * 3 - description asset ID
+  * **required** - set to 1 if asset is required or 0 if asset is optional (integer)
+  * **title**
+        * len (required) - maximum length of the text in the title element (integer)
+* **assets (data)**
+  * **id** - unique asset ID, assigned by exchange. Typically a counter for the array (integer):
+        *1 - image asset ID
+        * 2 - title asset ID
+        * 3 - description asset ID
+    * **data**
+      * **type** - type ID of the element supported by the publisher (integer). We support:
+            *1 - sponsored - sponsored By message where response should contain the brand name of the sponsor
+            * 2 - desc - descriptive text associated with the product or service being advertised
+        * **len** - maximum length of the text in the element’s response (integer)
+* **assets (img)**
+  * **id** - unique asset ID, assigned by exchange. Typically a counter for the array (integer):
+        *1 - image asset ID
+        * 2 - title asset ID
+        * 3 - description asset ID
+    * **required** - set to 1 if asset is required or 0 if asset is optional (integer)
+    * **img**
+      * **type** - type ID of the image element supported by the publisher. We support:
+            *1 - icon image (integer)
+            * 3 - large image preview for the ad (integer)
+        * **w** - width of the image in pixels, optional (integer)
+        * **h** - height of the image in pixels, optional (integer)
+
+##### DSA
```

Comment:
I appreciate you documenting that you support DSA. IMHO a link to https://github.com/InteractiveAdvertisingBureau/openrtb/blob/main/extensions/community_extensions/dsa_transparency.md should be enough and `dsa_supported: true` in the meta data